### ISSUE TITLE
Clipping planes don't adjust for 3D tileset scale

### ISSUE THREAD

**Issue by hpinkos** [AnalyticalGraphicsInc/cesium#5981]
This branch: https://github.com/AnalyticalGraphicsInc/cesium/tree/clip-planes-master

If you apply a scale to the model, the debug clipping planes no longer line up with where the model is clipped

**Comment by pjcozzi** [comment]
@ggetz can you please address this as part of putting the finishing touches on clipping planes?  The final "model" matrix for the tileset should take into account its model matrix and scale.

**Comment by ggetz** [comment]
I addressed this in [this branch](https://github.com/AnalyticalGraphicsInc/cesium/compare/clip-planes-master...ggetz:clipping-planes-terrain). I think this specific issue is about the sandcastle example, and how the displayed plane was not matching up with the actual clipping plane location on the tileset when scaled.

However, I did already add a "model" matrix property to specify any additional transforms for the clipping planes. 👍
